lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -27,11 +21,9 @@ export default function DashboardApp() {
       </div>
       <div className="flex-grow px-3">
         <div className="mx-auto max-w-6xl">
-          <StudentsActivityTable
-            assignment={assignment}
-            students={students.data ?? []}
-            loading={students.isLoading}
-          />
+          <Route path="/assignment/:assignmentId">
+            <StudentsActivity />
+          </Route>
```

Comment:
In follow-up PR we will be adding more routes here, each one of them rendering a different component corresponding to that page.